code/modules/clothing/modular_armor/tdf.dm

```diff
@@ -244,6 +244,34 @@
 		/obj/item/armor_module/storage/medical,
 	)
 
+/obj/item/clothing/suit/modular/tdf/robot
+	name = "\improper Clubs class heavy chassis"
+	desc = "Heavy armor plating designed for self mounting on TDF combat robotics. It has self-sealing bolts for mounting on robotic owners inside. Alt-Click to remove attached items. Use it to toggle the built-in flashlight."
+	species_exception = list(/datum/species/robot)
+	icon_state = "tdf_robot"
+	worn_icon_state = "tdf_robot"
+	soft_armor = list("melee" = 50, "bullet" = 70, "laser" = 75, "energy" = 60, "bomb" = 55, "bio" = 55, "fire" = 55, "acid" = 60)
```

Comment:
? Use defines, better yet jsut use the standard heavy armour define. Lasers are exclusive to tgmc, so the bonus laser armour doesn't even help at all against SOM.